bluepysnap/spike_report.py

```diff
@@ -194,3 +241,19 @@ def __getitem__(self, population_name):
     def __iter__(self):
         """Allows iteration over the different PopulationSpikeReports."""
         return iter(self._population)
+
+    def filter(self, group=None, t_start=None, t_stop=None):
+        """Returns a FilteredSpikeReport.
+
+        A FilteredSpikeReport is a lazy and cached object which contains the filtered data
+        from all the populations of a report.
+
+        Args:
+            group (None/int/list/np.array/dict): Get spikes filtered by group. See NodePopulation.
+            t_start (float): Include only frames occurring after this time.
```

Comment:
Same remarks about inclusion/exclusion of bounds.

Reply:
done